source/util.js

```diff
@@ -68,3 +69,7 @@ exports.getTagVersionPrefix = pMemoize(async options => {
 		return 'v';
 	}
 });
+
+exports.hostedGitInfo = repoUrl => {
+	return hostedGitInfo.fromUrl(repoUrl) || {};
+};
```

Comment:
```suggestion
exports.hostedGitInfo = repoUrl => hostedGitInfo.fromUrl(repoUrl) || {};
```

Reply:
Changed in 1c255d8a09765f13f8337d102e9d2d49ccf19fd6.